### PR TITLE
bump-version and update CHANGELOG

### PR DESCRIPTION
**Pull request by sateeshperi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,9 +10,10 @@
 - 🔄 Updated template to nf-core/tools v2.14.1 [#399](https://github.com/nf-core/methylseq/pull/399)
 - 🐛 fix sample/replicate merge [#401](https://github.com/nf-core/methylseq/pull/401)
 - 🐛 fix fasta reference usage [#403](https://github.com/nf-core/methylseq/pull/403)
-- 🐛 fix missing type field in nextflow_schema.json [#417](https://github.com/nf-core/methylseq/pull/417)
+- 🐛 fix missing type field in nextflow_schema.json #361 [#417](https://github.com/nf-core/methylseq/pull/417)
```
Comment (mashehu) [inline CHANGELOG.md:13]:
```suggestion
- 🐛 fix missing type field in nextflow_schema.json[#417](https://github.com/nf-core/methylseq/pull/417)
```
merge artifact?

**Review by mashehu — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (sateeshperi) [reply CHANGELOG.md:13]: #361 is the issue number

Comment (sateeshperi) [reply CHANGELOG.md:13]: I will let this be for this time @mashehu

Comment (mashehu) [reply CHANGELOG.md:13]: ah, sorry, didn't click through, my bad

**Pull request merged**